src/features/lookup_table/lookup_table_feature.py

```diff
@@ -21,15 +21,18 @@
     responses={**responses},
 )
 @create_response()
-def create_lookup(recipe_package: str, application: str, user: User = Depends(auth_w_jwt_or_pat)):
+def create_lookup(
+    application: str, recipe_package_paths: list[str] = Query(), user: User = Depends(auth_w_jwt_or_pat)
```

Comment:
The URL looks weird to have "recipe_package_paths=one_path,recipe_package_paths=another_path".
Think we should either have this as one comma separated string, or multiple parameters "recipe_package". 
More important that the parameters and sensibly named on the outside, than in the controller.

Reply:
It does not seem like comma separated list is supported in fastAPI without writing our own code to extract the list elements (https://github.com/tiangolo/fastapi/issues/50)

renaming the variable instead.